server/controller/accountsMangment/captains/addCaptain.js

```diff
@@ -0,0 +1,60 @@
+const { insertCaptain } = require('../../../database/queries/captain/addCaptain');
+
+const addCaptain = (req, res) => {
+  // console.log(8888, req.body, 88888888);
+  const {
+    name,
+    email,
+    phone,
+    address,
+    IDNumber,
+    licenceNumber,
+    status,
+    avatar,
+  } = req.body;
+
+  const captainInfo = {
+    name,
+    email,
+    phone,
+    address,
+    IDNumber,
+    licenceNumber,
+    status,
+    avatar,
+    type: 2,
+  };
```

Comment:
if you look to the table of `TUser` there is some column must be not null and some columns you need to add it for captain information like id number and driver license number
```
pk_i_id SERIAL PRIMARY KEY,
    s_name TEXT NOT NULL,
    s_mobile_number TEXT NOT NULL,
    s_email TEXT NOT NULL UNIQUE,
    b_status BOOLEAN NOT NULL,
    s_address TEXT NOT NULL,
    s_image TEXT NOT NULL,
    i_type INTEGER NOT NULL,
    dt_create_at DATE DEFAULT current_date,
    s_password TEXT NOT NULL,
    s_id_number INTEGER,
    s_attachment TEXT,
    s_driver_licence_number INTEGER,
```